pkg/internal/templates.go

```diff
@@ -166,45 +157,86 @@ func (w *wrapped{{.Name}}) checkCluster(ctx context.Context) (context.Context, e
 	}
 	return ctx, nil
 }
+`
+const patchTemplate = `
```

Comment:
I think we can join all these templates and add conditionals around them, no?

Reply:
(2) was the most important reason to split the templates. Couldn't see any other option for modifying specific verbs based on inputs if we collapse everything into one.